.github/workflows/doc-build.yml

```diff
@@ -35,7 +35,7 @@ env:
 jobs:
   doc-build-html:
     name: "Documentation Build (HTML)"
-    runs-on: ubuntu-22.04
+    runs-on: zephyr-runner-linux-x64-4xlarge
```

Comment:
it would be good to stick to ubuntu-22.04 as it's free -- was the reason here only to try and speed up the build, or something else?

Reply:
I saw PDF built was moved to this runner, and wanted to try with HTML. Build seems much faster (6-8min vs. 14-18min). If too expensive to run, we can continue using the free runner.

Reply:
Ya makes sense, we can give it a try. 
@nashif, FYI.

Reply:
yes, that looks good, every minute counts :)